reference-tests/src/main/java/net/consensys/linea/BlockchainReferenceTestOutcome.java

```diff
@@ -14,10 +14,17 @@
  */
 package net.consensys.linea;
 
-import java.util.List;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.ConcurrentSkipListSet;
+import java.util.concurrent.atomic.AtomicInteger;
 
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 
 @JsonPropertyOrder({"failedCounter", "successCounter", "modules"})
 public record BlockchainReferenceTestOutcome(
-    int failedCounter, int successCounter, List<ModuleToConstraints> modulesToConstraints) {}
+    AtomicInteger failedCounter,
+    AtomicInteger successCounter,
+    AtomicInteger disabledCounter,
+    AtomicInteger abortedCounter,
+    ConcurrentMap<String, ConcurrentMap<String, ConcurrentSkipListSet<String>>>
```

Comment:
Java records provide a way to model immutable data. You can of course mutate the data inside a record like update a map or increment a counter but that is an anti-pattern.